tests/clickhouse-test

```diff
@@ -673,6 +673,9 @@ class MergeTreeSettingsRandomizer:
         "primary_key_compress_block_size": lambda: random.randint(8000, 100000),
         "replace_long_file_name_to_hash": lambda: random.randint(0, 1),
         "max_file_name_length": threshold_generator(0.3, 0.3, 0, 128),
+        "sleep_before_commit_local_part_in_replicated_table_ms": threshold_generator(
```

Comment:
Doing it for all tests affects them in a bad way. 
Some sleeps in commit + sleeps in fetches + waiting on client, that consumes resources for nothing. The heavy inserting tests become too slow for nothing.
Will remove that from randomization since the solution is short and understandable.